packages/docs.nix

```diff
@@ -0,0 +1,26 @@
+{ pkgs, pname, ... }:
```

Comment:
Moved the main docs derivation here. The name of the file inside the packages directory gets passed into as `pname` which is why we inherit it later. This is a blueprint pattern.